apps/datahub/src/app/home/search/search-summary/search-summary.component.ts

```diff
@@ -18,6 +18,16 @@ export class SearchSummaryComponent {
     private searchService: SearchService
   ) {}
 
+  isOpen = false
+
```

Comment:
Maybe `SearchFilterComponent` would be more fitting for this component now?

Reply:
Good point!